agents/che-core-api-agent/src/main/java/org/eclipse/che/api/agent/server/launcher/AbstractAgentLauncher.java

```diff
@@ -172,4 +180,9 @@ void logAsErrorAgentStartLogs(Instance machine, String agentName, String logs) {
           machine.getNode().getHost());
     }
   }
+
+  @Override
+  public boolean blockMachineStart() {
```

Comment:
the name of the method looks odd for me (I was not expecting that is was returning something)

Reply:
@benoitf any suggestion?

Reply:
isSomething() ?

Reply:
`shouldBlockMachineStart()`?

Reply:
@l0rd yep I think it's clear as well

Reply:
but maybe add that it's in case of error.
shouldBlockMachineStartIfError() (but I agree it's a bit long :D )

Reply:
I changed to `shouldBlockMachineStartOnError()` :-)